src/tasks/LibraryBuilder/Templates/CMakeLists.txt.template

```diff
@@ -38,6 +38,10 @@ if(TARGETS_ANDROID)
         log
     )
     add_definitions(-DHOST_ANDROID=1)
+    add_definitions(-g)
+    add_definitions(-fPIC)
+    add_definitions(-fvisibility=hidden)
+    set(USE_GCC_ATOMIC_OPS 1)
```

Comment:
Right now this only applies to Android, didn't we want this applied to all?

Reply:
Unless I'm mistaken, we only apply these to android.